components/SearchResults/index.js

```diff
@@ -29,7 +32,10 @@ function SearchResults({
           : `No results for "${query}" in your word list.`}
       </StyledResultDisplay>
       {searchResults.length > 0 ? (
-        <EntriesContainer wordList={searchResults} />
+        <EntriesContainer
+          wordList={searchResults}
+          handleAddEntry={handleAddEntry}
+        />
       ) : (
         ""
       )}
```

Comment:
This would make it even easier to read:

```suggestion
{searchResults.length > 0 && (
        <EntriesContainer
          wordList={searchResults}
          handleAddEntry={handleAddEntry}
        />
)}
```